apps/heft/src/plugins/TypeScriptPlugin/TypeScriptBuilder.ts

```diff
@@ -121,13 +122,33 @@ export class TypeScriptBuilder extends SubprocessRunnerBase<ITypeScriptBuilderCo
   private _typescriptTerminal!: Terminal;
   private _emitCompletedCallbackManager: EmitCompletedCallbackManager;
 
+  private __tsCacheFilePath!: string;
   private _tsReadJsonCache: Map<string, object> = new Map<string, object>();
   private _cachedFileSystem: TypeScriptCachedFileSystem = new TypeScriptCachedFileSystem();
 
   public get filename(): string {
     return __filename;
   }
 
+  private get _tsCacheFilePath(): string {
+    if (!this.__tsCacheFilePath) {
+      const configHash: crypto.Hash = Tslint.getConfigHash(
```

Comment:
Seems like this should be moved out of the linter code and either into TypeScriptBuilder or into utilities.